docs/intro/ff-ui/toolbar.md

```diff
@@ -0,0 +1,234 @@
+
+### 2.7 What's New?
+
+We want to keep you updated with all the latest features and changes that are being made in FlutterFlow. Using this button, you can directly access the updates regarding what's new in FlutterFlow.
+
+### 2.8 Current Status/Known Issues
+
+We want to keep you informed about the current status of FlutterFlow and provide updates regarding any known issues. We aim to ensure transparency and keep you in the loop about the platform's performance and potential issues.
+
+### 2.9 Show/Hide Chat
+
+You can use this option to show or hide the chat button at the bottom right of the app builder.
+
+## 3. Keyboard shortcuts
+
+With keyboard shortcuts, you can perform common actions related to widgets and run your project in Test or Run mode with just a few keystrokes, saving you time and effort. Select this option to see all the shortcuts.
+
+![keyboard-shortcut.avif](imgs/keyboard-shortcut.avif)
+
+## 4. Command Palette
+
+The Command Palette makes it easy to find and use things in the App Builder. Open the Command Palette by clicking the search button or pressing Cmd/Ctrl + K. Search for anything, and to find out where it's used, click the right arrow – we'll show you every place. To directly access the item, just click on it.
+
+![command-palette.avif](imgs/command-palette.avif)
+
+## 5. Canvas size
+
+We allow you to visualize and design your app for various mobile, tablet, and desktop devices. The [canvas](#) size is defined as width (in pixels) x height (in pixels). This size is determined according to the device selected. To set a custom canvas size, click on the Canvas Size and enter the width and height (in pixels) you want.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/6TBNjYNfVDmV4NawxA4f?embed&show_copy_link=true"
+        title="Sharing a Project with a User"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+## 6. Project history
+
+Versions
+
+Versions enable you to save specific named states of your project. This function is particularly useful for recovery purposes. If you accidentally delete a page or component or change settings, you can easily revert to a saved version.
+
+Snapshots
+
+Snapshots are automatically saved as you build.
+
+:::info
+
+Users on the *Free* and *Standard* plans can access only one day of snapshots.
+
+:::
+
+### 6.1 Saving a version
```

Comment:
Add the 6.1 and 6.2 steps after you mention Versions at line 100 and describe it. Snapshots can be mentioned after the Version part is over.